products/compute/terraform.yaml

```diff
@@ -1435,6 +1455,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     properties:
       targetVpnGateway: !ruby/object:Overrides::Terraform::PropertyOverride
         resource: 'VpnGateway'
+      vpnGateway: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
I'd recommend a docs override here too to make it super clear that this comes from an ha_vpn_gateway resource and the other comes from a vpn_gateway resource.

Also I'm not convinced that `resource: 'Something'` here actually has any effect. Do you know if it does?

Reply:
It matters because the `vpnGateway` and `peerGcpGateway` are both ResourceRefs, and in `api.yaml` they both reference `VpnGateway` which is overridden via this file to point at `TargetVpnGateway`. 

Without this the tunnel was building its reference to `vpnGateway` using the `{stuff}/targetVpnGateways/{name}` form, which fails. Not sure why it doesn't just pull the self_link field directly from the vpn_gateway object

Reply:
oh of course, thanks! The reason it doesn't pull it directly from the object is, I think, because we accept both a name and a self link for it but send the self link back to the API, so we have to construct the self link if only the name is provided.